adapter/prometheus/prometheus.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package prometheus publishes metric values collected by the mixer for
+// ingestion by prometheus.
+package prometheus
+
+import (
+	"istio.io/mixer/adapter/prometheus/config"
+	"istio.io/mixer/pkg/adapter"
+)
+
+type (
+	factory struct {
+		adapter.DefaultBuilder
+
+		srv server
+	}
+
+	prom struct{}
+)
+
+var (
+	name = "prometheus.io/exporter" // TODO: better name?
```

Comment:
I'm not convinced that third-party integrations should be in the istio namespace. We wouldn't expect istio/gcp or something would we? To me istio/* are for bits where istio is doing everything (list checking, deny, writing to stdio).

Thoughts?